libs/ledger-live-common/src/families/polkadot/config.ts

```diff
@@ -7,6 +8,16 @@ export const polkadotConfig: Record<string, ConfigInfo> = {
       status: {
         type: "active",
       },
+      sidecar: {
+        url: getEnv("API_POLKADOT_SIDECAR"),
+        credentials: getEnv("API_POLKADOT_SIDECAR_CREDENTIALS"),
+      },
+      node: {
+        url: "https://polkadot-rpc.publicnode.com",
+      },
+      staking: {
+        electionStatusThreshold: getEnv("POLKADOT_ELECTION_STATUS_THRESHOLD"),
```

Comment:
it seems that this "electionStatusThreshold" is not used either if I don't make mistake? Should we keep it here?

Reply:
I suggest that we don't use environmental variables in config file. we can just write the default value here since with the firebase config framework, we can already change the config in real time.